packages/run-protocol/src/vaultFactory/vault.js

```diff
@@ -46,57 +43,55 @@ export const VaultPhase = /** @type {const} */ ({
 });
 
 /**
- * @type {{[K in VAULT_PHASE]: Array<VAULT_PHASE>}}
+ * @typedef {VaultPhase[keyof Omit<typeof VaultPhase, 'TRANSFER'>]} InnerPhase
+ * @type {{[K in InnerPhase]: Array<InnerPhase>}}
  */
 const validTransitions = {
-  [VaultPhase.ACTIVE]: [
-    VaultPhase.LIQUIDATING,
-    VaultPhase.TRANSFER,
-    VaultPhase.CLOSED,
-  ],
+  [VaultPhase.ACTIVE]: [VaultPhase.LIQUIDATING, VaultPhase.CLOSED],
   [VaultPhase.LIQUIDATING]: [VaultPhase.LIQUIDATED],
-  [VaultPhase.TRANSFER]: [VaultPhase.ACTIVE, VaultPhase.LIQUIDATING],
-  [VaultPhase.LIQUIDATED]: [],
+  [VaultPhase.LIQUIDATED]: [VaultPhase.CLOSED],
   [VaultPhase.CLOSED]: [],
 };
 
 /**
+ * @typedef {VaultPhase[keyof typeof VaultPhase]} OuterPhase
+ *
  * @typedef {Object} VaultUIState
  * @property {Amount<NatValue>} locked Amount of Collateral locked
  * @property {{run: Amount<NatValue>, interest: Ratio}} debtSnapshot Debt of 'run' at the point the compounded interest was 'interest'
  * @property {Ratio} interestRate Annual interest rate charge
  * @property {Ratio} liquidationRatio
- * @property {VAULT_PHASE} vaultState
+ * @property {OuterPhase} vaultState
  */
 
 /**
  *
  * @param {InnerVault | null} inner
  */
 const makeOuterKit = inner => {
+  /** @type {NotifierRecord<VaultUIState>} */
   const { updater: uiUpdater, notifier } = makeNotifierKit();
 
-  const assertActive = v => {
+  /** @returns true iff this outer vault is backed by an inner vault */
+  const assertOwned = v => {
```

Comment:
The terminology seems reversed here. The query is about the outer vault, which either owns or doesn't own the inner vault. It's not the one that might be 'owned'.

Why isn't 'active' a good term for this?

Reply:
> The query is about the outer vault

It's a little confusing but the param is just for the error message. The query is about the inner vault. 

> Why isn't 'active' a good term for this?

Because "active" is a particular state a vault can be in.  I have another PR that has another idea for the name so we can pick it up again in https://github.com/Agoric/agoric-sdk/pull/4743